### PR TITLE
THU/scenario logs on run success

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js [hunk 1]
```diff
@@ -13,24 +14,47 @@ const STORAGE_DASHBOARDS_ACCORDION_EXPANDED_KEY = 'dashboardsAccordionExpanded';
...
   };
 
+  const downloadLogsButton = useMemo(() => {
+    if (!hasRunBeenSuccessful) return null;
+
+    const triggerLogsDownload = (event) => {
+      event.stopPropagation();
+      downloadCurrentScenarioRunLogs();
+    };
+    const color = isDashboardSync ? 'primary' : 'inherit';
```
Comment (esasova) [inline src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js:36]: maybe we can define the color inside the component (the condition is rather simple to create a special `const` for) ?

**Review by esasova — APPROVED** [review]
Just one question but LGTM :+1:

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js (hunk 1 above)
Comment (csm-thu) [reply src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js]: Good idea :+1:

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.spec.js [hunk 2]
```diff
@@ -14,11 +19,26 @@ jest.mock('../index', () => ({
...
+};
+
 const Accordion = new AccordionTesting({ dataCy: 'dashboards-accordion' });
+const LogsDownloadButton = new ButtonTesting({ dataCy: 'successful-run-logs-download-button' });
 const getDashboard = () => screen.getByTestId('scenario-powerbi-report');
 
 describe('test Scenario dashboard accordion', () => {
   it('unfolds and folds scenario dashboard card', async () => {
+    setScenarioDashboardCardHook();
     customRender(<ScenarioDashboardCard />);
```
Comment (nborde-CSM) [inline src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.spec.js:42]: Same as below

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

File: src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.spec.js [hunk 3]
```diff
@@ -29,3 +49,19 @@ describe('test Scenario dashboard accordion', () => {
     await waitFor(() => expect(getDashboard()).not.toBeVisible());
   });
 });
+
+describe('display the logs download button only when necessary', () => {
+  it('is hidden when the scenario run has failed or has never been launched', () => {
+    setScenarioDashboardCardHook({ hasRunBeenSuccessful: false });
+    customRender(<ScenarioDashboardCard />);
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.spec.js:56]: IIRC, I had to set the custom hook value before rendering to properly render and test the desired behavior

**Pull request merged**